ssh_tunnel.go

```diff
@@ -23,6 +23,8 @@ type SSHTunnel struct {
 	close    chan interface{}
 }
 
+const MAX_CONNECTION_ATTEMPTS int = 10
```

Comment:
I don't want this to be a constant, rather it should be configurable. For example:

```go
tunnel := sshtunnel.NewSSHTunnel(...)

tunnel.MaxConnectionAttempts = 10

go tunnel.Start()
```

To keep with backwards compatibility, `0` should not allow any retry attempts.

Reply:
Updated the code with your suggestions.